node/src/command.rs

```diff
@@ -32,10 +28,9 @@ fn load_spec(
     Ok(match id {
         "pint-local" => Box::new(chain_spec::pint_local_config(para_id)),
         "pint-dev" => Box::new(chain_spec::pint_development_config(para_id)),
-        path => Box::new(
-            chain_spec::ChainSpec::from_json_file(std::path::PathBuf::from(path))
-                .unwrap_or(chain_spec::pint_development_config(para_id)),
-        ),
+        path => Box::new(chain_spec::ChainSpec::from_json_file(
+            std::path::PathBuf::from(path),
+        )?),
```

Comment:
This will occurs

```
Error: Input("Relay chain argument error: Invalid input: Error opening spec file: No such file or directory (os error 2)")
```

When we pass no `chainspec.json` or `--chain <id>`, personally think running `pint-local` by default is better

Reply:
I see, this was defaulted to `.unwrap_or(chain_spec::pint_development_config(para_id)),` which I agree is more convenient.